adaptor-testing-webapp/src/main/java/com/sample/testing/servlet/ServiceProvidersFactory.java

```diff
@@ -47,7 +47,7 @@
 {
     private static Class<?>[] RESOURCE_CLASSES =
     {
-        TestCaseService.class
+        ServiceProviderService1.class
```

Comment:
Yeah, this one is ugly. But we put it into the codegen so might as well have to live with it.